dlp/snippets/inspect_content.py

```diff
@@ -983,6 +985,67 @@ def inspect_image_file_all_infotypes(
 # [END dlp_inspect_image_all_infotypes]
 
 
+# [START dlp_inspect_image_listed_infotypes]
+def inspect_image_file_listed_infotypes(
+    project: str,
+    filename: str,
+    info_types: List[str],
+    include_quote=True,
+) -> None:
+    """Uses the Data Loss Prevention API to analyze strings for protected data
+    in image file for given info types.
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        filename: The path to the file to inspect.
+        info_types:
+        include_quote: Boolean for whether to display a quote of the detected
```

Comment:
... "to display a matching snippet of the detected" ...

It's not super obvious what quote is if you haven't used our API before.

Reply:
updated, Thank you for reviewing